packages/cli/README-source.md

```diff
@@ -801,27 +801,31 @@ This object holds the user's auth details and the data for the API requests.
 
 ### `bundle.inputData`
 
-`bundle.inputData` is user-provided data for this particular run of the trigger/search/create, as defined by the inputFields. For example:
+`bundle.inputData` is user-provided data for this particular run of the trigger/search/action, as defined by the `inputFields`. For example:
```

Comment:
I suggest we change `action` back to `create`. `create` is a more specific term in the CLI while `action` has a broader meaning and sometimes includes `search` and `create`.

Reply:
Makes sense, sorry about that! I'll get it fixed up.

Reply:
^ @megan-driscoll